src/NSubstitute.Analyzers.Shared/Extensions/ObjectExtensions.cs

```diff
@@ -0,0 +1,10 @@
+namespace NSubstitute.Analyzers.Shared.Extensions
+{
+    internal static class ObjectExtensions
+    {
+        public static T To<T>(this object source)
```

Comment:
Maybe document this, especially to mention it can throw. Perhaps rename to `Cast`?

Reply:
Changed to Cast and docs added